src/api/index.ts

```
@@ -1,18 +1,63 @@
-import returnFetch from 'return-fetch';
+import returnFetch, { FetchArgs } from 'return-fetch';
+import { ServerNotRespondingError } from '@/errors';
 
 const BASE_URL = process.env.NEXT_PUBLIC_BASE_URL;
+const ABORT_MS = Number(process.env.NEXT_PUBLIC_ABORT_MS);
 
-if (!BASE_URL) {
-  throw new Error('BASE_URL가 ENV에서 설정되지 않았습니다.');
+if (isNaN(ABORT_MS)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)